src/com/core/tictactoe/Board.java

```diff
@@ -39,10 +39,10 @@ String valueAtIndex(int index) {
     }
 
     boolean isWon() {
```

Comment:
There's duplicated logic between `isWon` and `winnerSign`, maybe `isWon` can use the result of `winnerSign` instead of duplicating.

Reply:
Done.